src/components/admin/Dialogs/ActionInfo.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import * as React from 'react';
+import Button from '@material-ui/core/Button';
+import {Typography} from '@material-ui/core';
+import {
+  Dialog,
+  DialogContent,
+  DialogContentText,
+  DialogActions,
+  DialogTitle,
+} from '@material-ui/core';
+
+type PropsType = {
+  action: any,
```

Comment:
using `any` in these places seems to be counter-productive, can we be more specific?

Reply:
I chose to leave some of the object properties as `any` because these are fake objects to test the front-end anyway. Once I start wiring the database to this part of the project, the user/action objects are going to have real schemas, so I'll add real `types` to the objects then - tldr: I didn't want to spend too much time defining object types for elements I know I'm going to throw away soon.

Reply:
ok